src/main/java/net/logstash/logback/stacktrace/ShortenedThrowableConverter.java

```diff
@@ -160,6 +165,11 @@ public class ShortenedThrowableConverter extends ThrowableHandlingConverter {
      */
     private boolean inlineHash;
 
+    /**
+     * True to use "\\n" sequence instead of {@link CoreConstants#LINE_SEPARATOR}
+     */
+    private boolean inline;
+
```

Comment:
Rather than having an `boolean inline` field, I think this should be a `String lineSeparator` field, initialized to `CoreConstants.LINE_SEPARATOR`.

Then `parseOptions` can call `setLineSeparator(INLINE_SEPARATOR)` when the `inline` option is present.

This will be slightly more flexible when configuring the converter programmatically or via xml.  And it will be more in-line (pun intended) with other classes that also have `lineSeparator` fields (e.g. `CompositeJsonEncoder`)

Reply:
ok thank you for the suggestion. I'll update it as soon as possible !

Reply:
done